numexpr/utils.py

```diff
@@ -152,7 +152,8 @@ def _init_num_threads():
     # actual number of threads used.
     if 'NUMEXPR_NUM_THREADS' in os.environ:
```

Comment:
Would be good to fix this one as well, and any others that blindly call int on an environmental variable.  Would it be more pythonic to try/except and then reraise (or warn) with an explanation?

Reply:
I think an empty string should be fine, so just ignoring it seems good to me